gen/calc.py

```diff
@@ -1328,6 +1389,13 @@ def calculate_fault_domain_detect_contents(fault_domain_detect_filename):
         'enable_windows_agents': 'false',
         'windows_dcos_install_path': 'C:\\d2iq\\dcos',
         'windows_dcos_var_path': 'C:\\d2iq\\dcos\\var',
+        'calico_network_cidr': '192.168.0.0/16',
```

Comment:
Unfortunately, we cannot calico configurations into only one blocker, because template analyzer does not support nested parameters.

Reply:
`calico_network_cidr` should be a mandatory option, but as it takes too many changes including other reops like TeamCity configuration and terraform module update, so we tracked this task as a blocker here https://jira.mesosphere.com/browse/DCOS-60734